ApiGateway/Mqtt/PointStateMessageHandler.cs

```diff
@@ -1,8 +1,7 @@
 namespace MRS.ApiGateway.Mqtt;
```

Comment:
Should MQTT be capitalised here?

Reply:
No, deliberately done as it is as then you can spot the change in word if it goes on for more than one (As in ApiGateway)